app/models/product.rb

```diff
@@ -23,4 +23,5 @@ class Product < ApplicationRecord
     .where("categories.name LIKE ?", "%#{key}%")
   end)
   scope :search_products, ->(key){where "products.name LIKE ?", "%#{key}%"}
+  scope :load_products_with_key, ->(key){where(id: key)}
```

Comment:
```suggestion
  scope :load_by_ids, ->(ids){where id: ids}
```
ri đi cậu, id thì ghi id chứ key là j